packages/camera/camera_android_camerax/lib/camera_android_camerax_platform_interface.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+import 'camera_android_camerax_method_channel.dart';
+
+/// The platform interface for the Android Camera implemented with CameraX.
+abstract class CameraAndroidCameraxPlatform extends PlatformInterface {
```

Comment:
This should be extending the `camera` plugin's platform interface class, not `PlatformInterface` directly.

I would suggest as a starting point for the Dart code in this plugin that you delete all the Dart files here, and instead make a copy of `android_camera.dart` from `camera_android`, and replace all the implementations with unimplemented calls, then fill them in over time.